servicex/servicex.py

```diff
@@ -230,10 +241,11 @@ def protected_list_objects(client: Minio, request_id: str):
 def get_data(selection_query: str, datasets: Union[str, List[str]],
              servicex_endpoint: str = 'http://localhost:5000/servicex',
              data_type: str = 'pandas',
-             image: str = 'sslhep/servicex_xaod_cpp_transformer:v0.2') \
+             image: str = 'sslhep/servicex_xaod_cpp_transformer:v0.2',
```

Comment:
Same question here; should we update to the v0.4 tag?

Reply:
Yes. I will do that in a separate PR as I move to the final version.